platform/linux-generic/test/pktio_ipc/pktio_ipc1.c

```diff
@@ -200,7 +200,11 @@ static int pktio_run_loop(odp_pool_t pool)
 		}
 
 		/* 3. emulate that pkts packets were received  */
-		odp_random_data((uint8_t *)&pkts, sizeof(pkts), 0);
+		ret = odp_random_data((uint8_t *)&pkts, sizeof(pkts), ODP_RANDOM_BASIC);
+		if (ret != sizeof(pkts)) {
+			LOG_ABORT("random failed");
```

Comment:
I think it's ok if odp_random_data() will write less data then requested in one shot.

Reply:
I do not want to have a loop around `odp_random_data` filling single int value (4 bytes). Feel free to add one, if you'd like to.

Reply:
Just note that previously there was no check for return value at all.